src/main/scala/io/citrine/lolo/learners/RandomForest.scala

```diff
@@ -68,8 +68,8 @@ case class RandomForest(
           numFeatures = numFeatures,
           minLeafInstances = minLeafInstances,
           maxDepth = maxDepth,
-          splitter = RegressionSplitter(randomizePivotLocation),
-          rng = rng
+          splitter = RegressionSplitter(randomizePivotLocation, rng = new Random(rng.nextLong())),
+          rng = new Random(rng.nextLong())
```

Comment:
It would be simpler to pass `rng` directly in both of these spots, but it seems like it is cleaner to split off two separate rngs for each piece. Am I overthinking this? @gregor-robinson

Reply:
Can you just construct the `rng` outside the match arm and pass it to both?

Reply:
Oh I am dumb, there already is an `rng` outside the match arm. I guess I don't see why we can't just pass that `rng` down?

Reply:
I'm probably over-thinking it and could just pass the `rng` down. But I'd like to check with Gregor